src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,68 @@
+.watch {
+  position: relative;
+
+  @include circle($watch-size);
+  border: $watch-border solid #000;
+  background-color: #f2f3f4;
+  margin: 50px auto;
+
+  &__minute-hand {
```

Comment:
you duplicate many styles in `minute/second-hand`, move them into common selector